module/Decision/src/Decision/Controller/FileBrowser/FileNode.php

```diff
@@ -36,6 +42,8 @@ public function __construct($kind, $fullPath, $name)
         $this->kind = $kind;
         $this->fullPath = $fullPath;
         $this->name = $name;
+        if ($kind==='dir') $this->extension = 'folder';
+        else $this->extension = strtolower(end(explode(".", $name)));
```

Comment:
All if else blocks should use brackets

Reply:
Changed in 3eea7fd4ced3cb3f1bcdaa008fd431af0f57ebfa